packages/documentation/copy/en/reference/Target Language Details.md

```diff
@@ -883,6 +882,8 @@ s = new Source(sequence={= new Array<number() =});
 
 Parameters and state variables in Rust are accessed on the `self` structure, as shown in [Parameter Declaration](/docs/handbook/parameters-and-state-variables#parameter-declaration).
 
+<span class="warning">**FIXME:** How to define and use array parameters and states?</span>
```

Comment:
This looks like it needs attention still?

Reply:
```suggestion
```
Since arrays are covered in the text above, I'm thinking this can just be removed?

Reply:
If not, please open an issue...